packages/upward-js/lib/resolvers/TemplateResolver.js

```diff
@@ -63,6 +63,7 @@ module.exports = class TemplateResolver extends AbstractResolver {
             providePromise
         ];
 
+        // eslint-disable-next-line prefer-const
```

Comment:
Instead of using this comment, these lines should probably be rewritten.

```es6
const [engineName, template, rootEntries] = await Promise.all(toResolve)
const engine = engineName || "mustache"
```

Reply:
Resolved in https://github.com/magento-research/pwa-studio/pull/1088/commits/c12f9179443d38f9c434dfbfda04872e2033dad6.